src/components/BaseInfoUpdatePage/BaseInfoUpdate.tsx

```diff
@@ -335,10 +335,10 @@ export const BaseInfoUpdate = (props: BaseInfoUpdateProps) => {
                         {isCurrentUser ? `Mes compétences` : `Compétences`}
                     </h2>
                     {isCurrentUser && (
-                        <p>
+                        <label className="fr-label fr-mb-3w" htmlFor="competence-editor">
```

Comment:
C'est pas un attribut for qu'il faut là plutôt ? (ping @freesteph)

Reply:
`for` est réservé en HS d'où l'astuce d'utiliser `htmlFor`. J'ignorais que c'était une [vraie propriété](https://developer.mozilla.org/en-US/docs/Web/API/HTMLLabelElement/htmlFor)

Reply:
c'est la syntaxe React mais une fois transformé en html, ça fera bien un for.